compose/dev/Dockerfile

```diff
@@ -3,14 +3,12 @@ FROM python:3.9-slim
 RUN apt-get -y update && apt-get install -y ffmpeg
 
 ENV PYTHONPATH /app
-ENV POETRY_VERSION=1.1.4
 
 COPY . /app
 WORKDIR /app
 
-RUN pip install "poetry==$POETRY_VERSION"
-RUN poetry config virtualenvs.create false
-RUN poetry install
+RUN pip install pipenv
```

Comment:
I recommend to specify the version here, or we'll eventually get broken build suddenly

Reply:
Good idea. I'm going to fix these comments this weekend 🚀 
Thanks for the advice.

Reply:
GJ